### DISCUSSION TITLE
AE Suggestion

### DISCUSSION THREAD

**Discussion by composition-cassidy** [otomad/om_midi#11] [Ideas]
Hey, I can't make a discussion post on the After Effects script's discussion tab but is it possible for the script to leave any gaps in the visual?

One more thing I want to edit on, for the Vegas script, is it possible to change the opacity from the MIDI? This would be really useful with Hi-hats and such

**Comment by otomad** [comment]
For After Effects script, I forgot to open the discussion...

For change the opacity. Yes, you guessed it, this will go live soon.

**Reply by composition-cassidy** [reply]
Hell yeah!! Thank you so much

**Comment by otomad** [comment]
> the script to leave any gaps in the visual

What does it mean?

**Reply by composition-cassidy** [reply]
Whenever I usually place visuals in After Effects, the visuals are always filled or left with no blank gaps. Is there a way to fix that?

**Reply by otomad** [reply]
Well I still don't quite understand what's mean, you can send me pictures or videos on Discord.

**Reply by composition-cassidy** [reply]
Oh, it's just like the Legato feature but for After Effects, is it possible?

**Reply by otomad** [reply]
Sorry I'm not sure if I understand it that way, can you provide a picture to illustrate this?

**Reply by otomad** [reply]
Is it like going from
![image](https://user-images.githubusercontent.com/56647156/185867535-78727b13-580c-41ec-80e4-496b6cf9aaf0.png)
to
![image](https://user-images.githubusercontent.com/56647156/185867782-29b49025-13f5-4a4e-b789-fc6ba38aa076.png)
?

I'm very sorry because I need to understand your needs more clearly.
